stdlib/TOML/test/values.jl

```diff
@@ -54,6 +52,30 @@ end
     @test testval("+1_000" , 1000  |> Int64)
     @test testval("-1_000" , -1000 |> Int64)
 
+    @test testval("0x6E"    ,0x6E|> UInt64)
+    @test testval("0x8f1e"  , 0x8f1e|> UInt64)
```

Comment:
```suggestion
    @test testval("0x6E" , 0x6E|> UInt64)
    @test testval("0x8f1e" , 0x8f1e|> UInt64)
```
Weird spacing if these lines aren't going to be aligned.